tests/install.sh

```diff
@@ -1,34 +1,33 @@
 #!/bin/bash
-
 #########################
 # Purpose: install the necessary software for integrating Autolab on Travis
 # Authors: Ankshit Jain, Kashyap Gajera, Prasad Talasila
 # Invocation: $bash install.sh
-#
+# Date: 05-Feb-2018
+# Previous Versions: 24-March-2017
 #########################
 
-npm install --quiet -g jshint 1>/dev/null
-npm install --quiet -g eslint 1>/dev/null
-npm install --quiet -g npm-check 1>/dev/null
-
+set -e
```

Comment:
It is worth adding the shell debug setting of `set -ex` at the beginning of all the shell scripts.

Reply:
`set -ex` lists all the commands executed one by one. Do we need that or should we stick with `set -e` for all the shell scripts?
This is the output of using set -ex
```
$ bash tests/install.sh
+mysql -e 'CREATE DATABASE Autolab;'
+mysql -e 'USE Autolab;'
+echo -e 'USE mysql;\nUPDATE user SET password=PASSWORD('\''root'\'') WHERE user='\''root'\'';\nFLUSH PRIVILEGES;\n'
+mysql -u root
+npm --quiet install --prefix main_server
```

Reply:
`set -ex` is better for our case.

Reply:
@prasadtalasila Please verify the updated commit. The wiki page is better updated when the changes to the testing structure is also included. That can be done once we agree on the implemented changes for issue #263. Also verify the travis build as `set -ex` introduces a number of things in the build log.